tests/test_kernel_pcovr.py

```diff
@@ -24,7 +24,12 @@ def __init__(self, *args, **kwargs):
 
         self.error_tol = 1e-6
 
-        self.X, self.Y = load_boston(return_X_y=True)
+        self.X, self.Y = get_dataset(return_X_y=True)
+
+        # for the sake of expedience, only use a subset of the dataset
+        idx = np.random.choice(len(self.X), 1000)
```

Comment:
we use above random_state, in this case one needs to use the state to do the random choice
```
self.random_state.choice(len(self.X), 1000)
```
otherwise one could use np.random.seed before(0), then this also works

Reply:
Remedied!